.github/workflows/rune_ubuntu_with_docker_and_crictl.yml

```diff
@@ -49,6 +49,12 @@ jobs:
         make package components=rune;
         dpkg -i rune_$RUNE_VERSION-1_amd64.deb;
         popd"
+    
+    - name: Build and install rune binary
```

Comment:
In the last step, install the rune package. This step is useless.

Reply:
Or you can delete the `Build and install rune deb` step and leave the `Build and install rune binary` step.
Change the name of `Build and install rune binary` to `Build and install binaries on ubuntu`.  In addition, do the same thing on rune_centos.yml.